normandy/conftest.py

```diff
@@ -11,3 +14,15 @@ def api_client():
     client = APIClient()
     client.force_authenticate(user=user)
     return client
+
+
+@pytest.yield_fixture()
+def django_cache(request):
```

Comment:
I tried to make a marker like `@pytest.market.django_db` (which, btw, just turns around and uses a fixture). I couldn't get it to work though. I can learn more about markers and try to get it working that way.